recipes/ngseqbasic/meta.yaml

```diff
@@ -0,0 +1,38 @@
+
+package:
+  name: ngseqbasic
+  version: 2.0.1
+
+source:
+  fn: NGseqBasic.tar.gz
+  url: https://github.com/telenius/NGseqBasic/raw/master/NGseqBasic.tar.gz
+  md5: 3992a5c4ed260e548db9b47cad3009be
+
+build:
+  skip: True # [osx]
```

Comment:
This software is mainly targeted to Linux clusters. Moreover:
* The developer won't offer support for OSX
* OSX users might try the containerised version automatically provided with conda, if needs be